test/spoof/spoof.go

```diff
@@ -274,7 +274,7 @@ func (sc *SpoofingClient) logZipkinTrace(spoofResp *Response) {
 
 	zipkinTraceEndpoint := zipkin.ZipkinTraceEndpoint + traceID
 	// Sleep to ensure all traces are correctly pushed on the backend.
-	time.Sleep(5 * time.Second)
+	time.Sleep(30 * time.Second)
```

Comment:
30 seconds is very high. We debated a lot when we set it to 5 secs because it adds latency. So if I understand it correctly you are saying that after 5 secs you are not seeing traces and a higher wait time causes you see traces?

Reply:
Yes, that is correct. For 5 seconds, I see traces half of the time. I can try to run a few more tests to see if we can bring down 30 second sleep time. 

Note that I was mainly testing if zipkin logs are printed for success responses by commenting out the server error check. As I could not figure out a nice way to always force a 500/600 response. However, for cases where it did fail with 504 error, I noticed the log to be much shorter than success. It could be the 5 seconds is not enough for success, but may be enough for server error. Please let me know if there's a good way to test this hunch and confirm this PR is not required.

Reply:
Ya I feel that success based retrieval is overwhelming the system. Is the zipkin setup in-memory or elastic search backed?

Reply:
It is elastic search backed.